install/kubernetes/cilium/templates/cilium-nodeinit-daemonset.yaml

```diff
@@ -242,6 +251,15 @@ spec:
                   docker kill "${container_id}" || true
                 done
               else
+                # Wait until cilium-agent is up and running which means the cni
+                # configuration and cni plugin will be deployed.
+                echo "Waiting for Cilium agent to be up and running"
+                until PATH="${PATH}:/home/kubernetes/bin" crictl ps | grep cilium-agent | grep -q Running; do echo -n "." && sleep 1; done
+                # Sleep 10 seconds just to be safe since the configuration and
+                # cni binary will not be deployed immediately as the
+                # cilium-agent container starts.
+                sleep 10s
+                echo ""
                 # COS-beta (with containerd). Some versions of COS have crictl in /home/kubernetes/bin.
                 for f in `find /var/lib/cni/networks/ -type f ! -name lock ! -name last_reserved_ip.0`; do PATH="${PATH}:/home/kubernetes/bin" crictl stopp "$(sed 's/\r//;1q' $f)" || true; done
```

Comment:
Not related to this PR, but does crictl really have a `stopp` command or is that meant to say `stop`?

Reply:
^ Answer: Yes, this is expected:

```
# crictl help  | grep stopp
   stopp               Stop one or more running pods
```